scaleway/resource_tem_domain.go

```diff
@@ -140,6 +170,12 @@ func resourceScalewayTemDomainRead(ctx context.Context, d *schema.ResourceData,
 	_ = d.Set("last_error", domain.LastError)
 	_ = d.Set("spf_config", domain.SpfConfig)
 	_ = d.Set("dkim_config", domain.DkimConfig)
+	_ = d.Set("smtp_host", "smtp.tem.scw.cloud")
```

Comment:
Why is it a static string? Could it be read from the API instead of being hardcoded?

Reply:
There is no API to fetch this value. Same for ports.